### PR TITLE
Issue #15 Changing the script names to reflect new project name

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

### CODE REVIEW

**Comment by praveenkumar** [comment]
We also need to make changes to below location instead only changing the filename.

- https://github.com/praveenkumar/osp4/blob/master/libvirt/minishift_libvirt.sh#L262

**Comment by LalatenduMohanty** [comment]
> We also need to make changes to below location instead only changing the filename.

Ahh, missed it. Will fix it.

File: build.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Tarball content for the Linux host (using libvirt)
     - These disk images have all container images part of the disk. So user do not need to fetch container images from the internet.
 - Add the `kubeconfig` and `kubeadmin-password` which is generated by installer.
 - Add `master-private-key` which can be used to `ssh` to master or worker node.
-- Add `minishift.sh` shell script which can consume those bits and start cluster.
+- Add `crc_<hypervisor name>.sh` shell script which can consume those bits and start cluster.
```
Comment (praveenkumar) [inline build.md]: :+1:

File: libvirt/crc_libvirt.sh [hunk 2]
```diff
@@ -259,7 +259,7 @@ EOF
 </domain>
 EOF
     sudo virsh define ./test1-worker-0-98nsr.xml
-    echo "Cluster created successfully use './minishift.sh start' to start it"
+    echo "Cluster created successfully use '$0 start' to start it"
```
Comment (praveenkumar) [inline libvirt/crc_libvirt.sh]: :+1:

File: virtualbox/crc_virtualbox.sh [hunk 3]
```diff
@@ -20,7 +20,7 @@ prerequisites()
     if pgrep -x "minishift" > /dev/null
     then
         echo "Minishift is running which create issue with coredns"
-        echo "Stop the minishift or delete"
+        echo "Stop or delete the minishift instance"
```
Comment (praveenkumar) [inline virtualbox/crc_virtualbox.sh]: :+1:

**Pull request merged**